### ISSUE TITLE
TypeError on Pool Library Plates: data.join is not a function

### ISSUE THREAD

**Issue by wasade** [jdereus/labman#466]
To recreate:

1) select Pools -> Pool Library Plates
2) select a plate type

Offending line of code is [here](https://github.com/jdereus/labman/blob/master/labcontrol/gui/templates/library_pooling.html#L377).

<img width="1700" alt="Screen Shot 2019-04-17 at 6 19 11 PM" src="https://user-images.githubusercontent.com/474290/56330589-8728ad00-613d-11e9-8750-1cec2803b24a.png">

**Comment by wasade** [comment]
This error is appearing now on a few pages, including "Plate -> Prepare Amplicon Libraries" and is no longer listing plates. It seems as though something about the state of my database is off, but there isn't an indication anywhere as to what that might be. Other plate listings (e.g., "Plates -> List") works as expected and has plenty of plates across the different types. 

I blew away my test database and was able to recreate on a fresh database. My repository appears clean (see below), this suggests `master` is broken.

@AmandaBirmingham @antgonza @charles-cowart, would it be possible for an independent verification?

```bash
(base) 18:32:24 (dtmcdonald@here):labcontrol@master$ git pull upstream master
From github.com:jdereus/labman
 * branch            master     -> FETCH_HEAD
Already up to date.
(base) 18:33:01 (dtmcdonald@here):labcontrol@master$ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
(base) 18:33:05 (dtmcdonald@here):labcontrol@master$ git log | head
commit 8eb173b170b0e663771c46e5e96890658615c4b7
Author: charles-cowart <42684307+charles-cowart@users.noreply.github.com>
Date:   Wed Apr 17 14:31:01 2019 -0700

    Fixes #284 (#462)

    Adding Help menu option, with link to https://github.com/jdereus/labman/blob/master/docs/index.rst

commit ecb73d3cda87271556802a68608fca081b06c664
Merge: 9eeed01 c2ed17f
```

**Comment by AmandaBirmingham** [comment]
@wasade Not to worry!  I actually fixed this today during my work on #351 but have not put into PR yet :)  

This error was introduced by https://github.com/jdereus/labman/pull/412, which added plate creation dates to the data returned by the PlateListHandler (/plate_list) and updated  plate_list.html to expect this new date but did not update the javascript on the 7 other pages that also consume /plate_list results to expect an additional field.  The reason this error didn't start occurring in all 7 of those pages is that the javascript to parse the results of /plate_list is ALMOST but not EXACTLY the same in those 7 places (ER. MA. GERD :).  These various issues should all cease to exist very soon :)

**Comment by wasade** [comment]
Ah okay, thank you! Am quite excited for #351! Are you alright leaving this open until that PR is merged?

**Comment by AmandaBirmingham** [comment]
Closed by PR #468
